README.md

```diff
@@ -5,31 +5,24 @@
 
 # InputStream Adaptive add-on for Kodi
 
-This is an adaptive file addon for kodi's new InputStream Interface.
+This is a [Kodi](https://kodi.tv) input stream add-on wich acts as a demuxer for segmented, multi-bitrate internet streams. The most common streaming protocols such as MPEG-DASH, HLS and Microsoft Smooth Streaming are supported.
 
-- this addon is part of the official kodi repository and part of each kodi installation
-- configure the addon by adding URL prefixes wich are allowed to be played by this addon
-- create a .strm file / or addon with passes an url with sets inputstream.adaptive.manifest_type to either "mpd", "ism" or "hls" and open the strm file in kodi
-- or write an addon wich passes an mpd or ism manifest file to kodi
+The add-on also has support for DRM protected streams, such as Google Widevine, Microsoft PlayReady and others, for some of them are only available on specific operating systems.
 
-##### Examples:
-1.) mpd dash example with one video and one audio stream
-- Force inputstream.mpd using a property in strm file: #KODIPROP:inputstream=inputstream.adaptive
-- Select the tye of the manifest using a property in strm file: #KODIPROP:inputstream.adaptive.manifest_type=mpd
-- URL to paste into strm file: http://download.tsi.telecom-paristech.fr/gpac/DASH_CONFORMANCE/TelecomParisTech/mp4-live/mp4-live-mpd-AV-BS.mpd
+The add-on can be used from all video add-ons by appropriately configuring the *ListItem* properties to instruct Kodi to play the content by using this add-on.
+For test purposes is possible create the STRM files, where you can add the URL for the test stream to be played and the parameters to instruct Kodi to play the content by using this add-on.
 
-2.) mpd dash example with one video and multiple audio streams
-- Force inputstream.mpd using a Property in strm file: #KODIPROP:inputstream=inputstream.mpd
-- Select the tye of the manifest using a property in strm file: #KODIPROP:inputstream.adaptive.manifest_type=mpd
-- URL to paste into strm file: http://rdmedia.bbc.co.uk/dash/ondemand/testcard/1/client_manifest-events-multilang.mpd
+*Please refer to the [Wiki pages](https://github.com/xbmc/inputstream.adaptive/wiki), to read detailed instructions in how to integrate it in your add-on, how to make tests, how to build it and many other information.*
 
-##### Decrypting:
+## Features in WIP
+
+#### Decrypting:
 Decrypting is not implemented. But it is prepared!
```

Comment:
Doesn't have to be part of this PR but thought I'd mention anyway:

`Decrypting is not implemented. But it is prepared!`

This I think was relevant back in the early days of this add-on where wvdecrypter wasn't bundled with the add-on and even at one point only existed in liberty-developer's fork - I suggest this whole paragraph is removed or rewritten, perhaps mentioning that most commonly Widevine CDM (not supplied) is required.

Reply:
now that i read again you have right i will remove all this point